sonic_data_client/non_db_client.go

```diff
@@ -332,7 +333,107 @@ func (c *NonDbClient) String() string {
 
 // To be implemented
```

Comment:
Do you mind to update comment for this function for code readability? Thanks

Reply:
Ack